docs/src/development.md

```diff
@@ -125,6 +125,12 @@ ci/pip-install.sh
 python src/quality_time_notifier.py
 ```
 
+##### Running the proxy component
+
+The `proxy` component is mapped to the `www` service in the docker compose file.
+This container runs an unprivileged version of Nginx on port 8080, so it does not require additional capabilities.
+Because the upstream container assumes it is internally running on port 8080, this is not configurable.
```

Comment:
What's the "upstream container"? Did you mean "base image"?

Reply:
Indeed, I changed the docs